src/main/java/org/teamresistance/frc/Gyro3D.java

```diff
@@ -0,0 +1,42 @@
+package org.teamresistance.frc;
+
+import com.kauailabs.navx.frc.AHRS;
+import edu.wpi.first.wpilibj.SPI;
+import org.strongback.components.AngleSensor;
+
+/**
+ * @author Shreya Ravi
+ */
+public class Gyro3D extends AHRS implements AngleSensor {
+
+  public Gyro3D(SPI.Port port) {
+    super(port);
+  }
+
+
+  @Override
+  public double getAngle() {
+    return getYawVal();
+  }
+
+  private double normalize(double angle) {
+    return ((angle % 360) + 360) % 360;
+  }
+
+  public double getYawVal() {
```

Comment:
This method becomes obsolete when applying the decision above, as we'll just use `getAngle`. The others will more idiomatically become `getRoll` and `getPitch` as the names no longer conflict with the AHRS.